docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -0,0 +1,27 @@
+# Install Cluster-api-provider-metal3
+
+You can either use clusterctl (recommended) to install Metal³ infra provider
+or kustomize and directly install manifests yourself. Please keep in mind that
+Baremetal Operator and Ironic are decoupled from CAPM3 and will not be installed
+when the provider is initialized. As such, you need to install them yourself.
+
+## Prerequisites
+
+1. Install `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
+2. Install `kustomize`, refer to official instructions [here](https://kubectl.docs.kubernetes.io/installation/kustomize/).
```

Comment:
I'm guessing this is here for installing CAPM3 "manually", but there are no instructions for this yet? Maybe we could add something to make it a bit clearer, like a section "With kustomize" and then just TODO in there? Or perhaps link to the repo if there are instructions already there.

Reply:
Ops, indeed I am missing manual installation. Let me add it quickly

Reply:
@lentzi90 updated, PTAL. Thanks